site/docs/sample-queries/aws_ec2.md

```diff
@@ -35,13 +36,10 @@ INSERT INTO instance (ami, instance_type, tags)
 INSERT INTO instance_security_groups (instance_id, security_group_id) SELECT
   (SELECT id FROM instance WHERE tags ->> 'name' = 'i-2'),
   (SELECT id FROM security_group WHERE group_name='default' AND vpc_id = (SELECT id FROM vpc WHERE is_default = true));
+SELECT iasql_commit();
 ```
 
-Apply changes
-
-```sql
-SELECT * FROM iasql_apply();
-```
+The `iasql_begin()` and `iasql_commit()` functions are IaSQL RPCs that are used to start and then end a transaction. We use those two functions to push changes to the cloud.
```

Comment:
The transaction block doesn't appear to be necessary here.

Reply:
There's a `SELECT * FROM instance` call after this. That'll not have the cloud-provided fields and I think it's better for the UX to wrap the changes here in a transaction. But I've changed the helper sentence to a more informative one.